dist/recipe.nsi

```diff
@@ -47,7 +47,7 @@
 ;--------------------------------
 ;Folder selection page
  
-  InstallDir "$PROGRAMFILES64\graylog\collector-sidecar"
+  InstallDir "$PROGRAMFILES64\Graylog\collector-sidecar"
  
```

Comment:
Changing the installation path needs to be handled in case a user is updating the software